src/components/SideBar/SideBar_Item/SideBarItem.jsx

```diff
@@ -0,0 +1,40 @@
+import { useState, useContext } from 'react';
+import './styles.scss';
+import { ProductsContext } from '../../../ProductsContext';
+
+const SideBarItem = ({ item }) => {
+  const [isOpen, setIsOpen] = useState(false);
+  const productContext = useContext(ProductsContext);
+
+  if(item.childrens) {
+    return (
+      <div className="sidebar-wrapper" >
+        <li className={item.title === productContext.currentLink ? "sidebar-item sidebar-active" : "sidebar-item"}  onClick={() => {
+          setIsOpen(true);
+          productContext.setCurrentLink(prev => prev = item.title);
```

Comment:
```suggestion
          productContext.setCurrentLink(item.title);
```
K gán theo kiểu `prev = item.title` nhé